### PR TITLE
chore(husky, pnpm v7): bump husky to work with pnpm 7

### PR DESCRIPTION
**Pull request by aiheon**
### Proposed Changes

https://coveord.atlassian.net/browse/UITOOL-655

Same as https://github.com/coveo/admin-ui/pull/4543/ but for Plasma 👀 

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

**Review by gdostie — APPROVED** [review]

File: packages/website/docs/TextArea.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ If the input can be left blank, the tag “(Optional)” must appear next to the
 
 ## Variants
 
-| **Type**      | **Purpose** |
-| --------- | --------- |
-| **Fixed size** | By default, the size of the text area input box is fixed. A scrollbar automatically appears if text overflows the alloted space. |
+| **Type**           | **Purpose**                                                                                                                                                                                                                                                                                                                                                                                                         |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```
Comment (mikedidomizio) [inline packages/website/docs/TextArea.md:26]:
```suggestion
| - | - |
```

Although I already see it's common within the project, if we do the suggestion we don't have to keep it updated and it'll adjust when we make changes.  Having it all line up raw is nice but the actual content is all on one single line and not very readable as it is.  It's just one more thing to maintain that seems unnecessary, for instance [Toast.md](https://raw.githubusercontent.com/coveo/plasma/master/packages/website/docs/Toast.md) is not aligned anymore, but it never had to be aligned in the first place.  

(Plus I think it looks better than 100+ dashes).

**Review by mikedidomizio — APPROVED** [review]

**Review by toofff — APPROVED** [review]

File: packages/website/docs/TextArea.md (hunk 1 above)
Comment (aiheon) [reply packages/website/docs/TextArea.md:26]: It's indeed from the linter, I had to test the script to make sure it's still working with pnpm 7 🙈

Comment (mikedidomizio) [reply packages/website/docs/TextArea.md:26]: Okay.  I still like the idea of do it once, forget and not require continuous line changes but if it's automated well then my maintenance point is no longer relevant

Comment (aiheon) [reply packages/website/docs/TextArea.md:26]: Yeah I hope we won't have to change it again 🤞 😅 
Feels like .md is design by robots for robots 😆

**Pull request merged**